ansible/files/Makefile

```diff
@@ -35,5 +35,13 @@ darkk_weechat.cert: darkk_weechat.key pusher_ca.cert pusher_ca.key.vault
 %_ca.key.vault:
 	openssl genrsa 2048 | ../vault encrypt --output $@
 
+
 %.key:
 	touch $@ && chmod 600 $@ && openssl genrsa 2048 >$@
+
+%_user.key.vault:
+	openssl genrsa 2048 | ../vault encrypt --output $@
+
+postgres_amsrepl_user.cert: postgres_amsrepl_user.key.vault postgres_ca.cert
+	openssl req -new -key <(../vault view postgres_amsrepl_user.key.vault) -sha256 -days 3650 -subj '/O=OONI/OU=AMS Replica User/CN=amsrepl' | \
+	openssl x509 -req -sha256 -days 3650 -CA postgres_ca.cert -CAkey <(../vault view postgres_ca.key.vault) -set_serial 0x`head -c 16 /dev/urandom | xxd -plain` -out $@
```

Comment:
I figured that since that CA is already going to be distributed to all the pg instances we may as well use the same one for also signing client certificates.

Do you think that is a bad idea?